beets/autotag/match.py

```diff
@@ -447,6 +447,12 @@ def tag_album(items, search_artist=None, search_album=None,
             search_artist, search_album = cur_artist, cur_album
         log.debug(u'Search terms: {0} - {1}', search_artist, search_album)
 
+        extra_tags = None
+        if config['musicbrainz']['extra_tags']:
+            fields = {u'year', u'catalognum', u'country', u'media', u'label'}
```

Comment:
We could even make the configuration option a list of fields that is by default empty. Then the user could specify exactly the set of fields to include, which could help with experimentation.